test/input/closeness_centrality_unweighted_functor.gt

```diff
@@ -0,0 +1,49 @@
+element Vertex end
+element Edge end
+
+const edges : edgeset{Edge}(Vertex, Vertex) = load ("../../test/graphs/test.el");
+const vertices : vertexset{Vertex} = edges.getVertices();
+
+func updateEdge[checked_local: vector{Vertex}(int)](src : Vertex, dst : Vertex)
+     checked_local[dst] = checked_local[src] + 1;
+end
+
+func toFilter[checked_local: vector{Vertex}(int)](v : Vertex) -> output : bool
+     output = checked_local[v] == -1;
+end
+
+func main()
+
+    var checked_local : vector{Vertex}(int) = -1;
```

Comment:
We have to be careful while moving declarations local. This should work fine here but might break stuff for the GPU backend. Since the arrays need host and device copies, we rely on them being global and create copies for global declarations only. For now this is OK, but we should discuss how to generalize this.

Reply:
Ah I see. Yep let's do that.